harbor-cluster/templates/harborcluster.yaml

```diff
@@ -7,16 +7,17 @@ metadata:
 
 spec:
   version: {{ .Values.version }}
-  logLevel: info
+  logLevel: debug #info
```

Comment:
Need to be parametrized in as helm value.

Reply:
ok